test/opt/inst_bindless_check_test.cpp

```diff
@@ -27,178 +27,12 @@ namespace {
 
 using InstBindlessTest = PassTest<::testing::Test>;
 
-static const std::string kOutputDecorations = R"(
```

Comment:
This file will be much easier to review with whitespace ignored. I removed a lot of indentation from the embedded SPIRV disassembly because it is too hard to maintain.